roles/ceph-defaults/tasks/check_socket.yml

```diff
@@ -0,0 +1,21 @@
+---
+# These checks are used to avoid running handlers at initial deployment.
+- name: check for a ceph socket
+  shell: |
+    {{ docker_exec_cmd }} bash -c 'stat {{ rbd_client_admin_socket_path }}/*.asok > /dev/null 2>&1'
```

Comment:
Did you mean to have ``{{ docker_exec_cmd }}`` here? It's undefined unless a ``containerized_deployment`` is ``True``.